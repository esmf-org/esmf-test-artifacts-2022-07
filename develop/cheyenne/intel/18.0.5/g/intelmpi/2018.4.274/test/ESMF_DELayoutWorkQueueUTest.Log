build time -- 2022-07-14 03:03:59.593754
20220714 033140.570 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 033140.570 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220714 033140.570 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220714 033140.570 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220714 033140.570 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220714 033140.570 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 033140.570 INFO             PET0 Running with ESMF Version   : v8.4.0b04
20220714 033140.570 INFO             PET0 ESMF library build date/time: "Jul 14 2022" "02:55:57"
20220714 033140.570 INFO             PET0 ESMF library build location : /glade/scratch/dunlap/esmf-testing/intel_18.0.5_intelmpi_g_develop
20220714 033140.570 INFO             PET0 ESMF_COMM                   : intelmpi
20220714 033140.570 INFO             PET0 ESMF_MOAB                   : enabled
20220714 033140.570 INFO             PET0 ESMF_LAPACK                 : enabled
20220714 033140.570 INFO             PET0 ESMF_NETCDF                 : enabled
20220714 033140.570 INFO             PET0 ESMF_PNETCDF                : disabled
20220714 033140.570 INFO             PET0 ESMF_PIO                    : enabled
20220714 033140.570 INFO             PET0 ESMF_YAMLCPP                : enabled
20220714 033140.571 INFO             PET0  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
20220714 033140.571 INFO             PET0  NUMBER_OF_PROCESSORS           4
20220714 033140.571 INFO             PET0  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
20220714 033140.572 INFO             PET0  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
20220714 033140.572 INFO             PET0  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
20220714 033140.572 INFO             PET0  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
20220714 033146.601 INFO             PET0  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
20220714 033146.601 INFO             PET0  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
20220714 033146.601 DEBUG            PET0 vmkt_create()#198 Pthread: service=0 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 20971520 bytes
20220714 033146.602 DEBUG            PET0 vmkt_create()#198 Pthread: service=0 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 20971520 bytes
20220714 033146.602 DEBUG            PET0 vmkt_create()#198 Pthread: service=0 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 20971520 bytes
20220714 033146.602 DEBUG            PET0 vmkt_create()#198 Pthread: service=0 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 20971520 bytes
20220714 033146.616 INFO             PET0  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
20220714 033149.158 INFO             PET0  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
20220714 033149.158 INFO             PET0  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
20220714 033149.158 INFO             PET0  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
20220714 033149.158 INFO             PET0 Finalizing ESMF
20220714 033140.570 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 033140.570 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220714 033140.570 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220714 033140.570 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220714 033140.570 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220714 033140.570 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 033140.570 INFO             PET1 Running with ESMF Version   : v8.4.0b04
20220714 033140.570 INFO             PET1 ESMF library build date/time: "Jul 14 2022" "02:55:57"
20220714 033140.570 INFO             PET1 ESMF library build location : /glade/scratch/dunlap/esmf-testing/intel_18.0.5_intelmpi_g_develop
20220714 033140.570 INFO             PET1 ESMF_COMM                   : intelmpi
20220714 033140.571 INFO             PET1 ESMF_MOAB                   : enabled
20220714 033140.571 INFO             PET1 ESMF_LAPACK                 : enabled
20220714 033140.571 INFO             PET1 ESMF_NETCDF                 : enabled
20220714 033140.571 INFO             PET1 ESMF_PNETCDF                : disabled
20220714 033140.571 INFO             PET1 ESMF_PIO                    : enabled
20220714 033140.571 INFO             PET1 ESMF_YAMLCPP                : enabled
20220714 033140.571 INFO             PET1  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
20220714 033140.571 INFO             PET1  NUMBER_OF_PROCESSORS           4
20220714 033140.571 INFO             PET1  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
20220714 033140.572 INFO             PET1  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
20220714 033144.600 INFO             PET1  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
20220714 033144.600 INFO             PET1  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
20220714 033146.601 INFO             PET1  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
20220714 033146.601 INFO             PET1  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
20220714 033146.601 DEBUG            PET1 vmkt_create()#198 Pthread: service=1 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 4194304 bytes
20220714 033146.601 DEBUG            PET1 vmkt_create()#198 Pthread: service=1 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 4194304 bytes
20220714 033146.616 INFO             PET1  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
20220714 033149.092 INFO             PET1  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
20220714 033149.092 INFO             PET1  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
20220714 033149.092 INFO             PET1  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
20220714 033149.092 INFO             PET1 Finalizing ESMF
20220714 033140.569 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 033140.569 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220714 033140.569 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220714 033140.569 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220714 033140.569 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220714 033140.569 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 033140.569 INFO             PET2 Running with ESMF Version   : v8.4.0b04
20220714 033140.569 INFO             PET2 ESMF library build date/time: "Jul 14 2022" "02:55:57"
20220714 033140.569 INFO             PET2 ESMF library build location : /glade/scratch/dunlap/esmf-testing/intel_18.0.5_intelmpi_g_develop
20220714 033140.569 INFO             PET2 ESMF_COMM                   : intelmpi
20220714 033140.570 INFO             PET2 ESMF_MOAB                   : enabled
20220714 033140.570 INFO             PET2 ESMF_LAPACK                 : enabled
20220714 033140.570 INFO             PET2 ESMF_NETCDF                 : enabled
20220714 033140.570 INFO             PET2 ESMF_PNETCDF                : disabled
20220714 033140.570 INFO             PET2 ESMF_PIO                    : enabled
20220714 033140.570 INFO             PET2 ESMF_YAMLCPP                : enabled
20220714 033140.571 INFO             PET2  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
20220714 033140.571 INFO             PET2  NUMBER_OF_PROCESSORS           4
20220714 033140.571 INFO             PET2  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
20220714 033140.572 INFO             PET2  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
20220714 033146.601 INFO             PET2  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
20220714 033146.601 INFO             PET2  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
20220714 033146.601 INFO             PET2  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
20220714 033146.601 INFO             PET2  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
20220714 033146.601 DEBUG            PET2 vmkt_create()#198 Pthread: service=1 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 4194304 bytes
20220714 033146.601 DEBUG            PET2 vmkt_create()#198 Pthread: service=1 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 4194304 bytes
20220714 033146.616 INFO             PET2  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
20220714 033149.122 INFO             PET2  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
20220714 033149.123 INFO             PET2  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
20220714 033149.123 INFO             PET2  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
20220714 033149.123 INFO             PET2 Finalizing ESMF
20220714 033140.570 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 033140.570 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220714 033140.570 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220714 033140.570 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220714 033140.570 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220714 033140.570 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220714 033140.570 INFO             PET3 Running with ESMF Version   : v8.4.0b04
20220714 033140.570 INFO             PET3 ESMF library build date/time: "Jul 14 2022" "02:55:57"
20220714 033140.570 INFO             PET3 ESMF library build location : /glade/scratch/dunlap/esmf-testing/intel_18.0.5_intelmpi_g_develop
20220714 033140.570 INFO             PET3 ESMF_COMM                   : intelmpi
20220714 033140.571 INFO             PET3 ESMF_MOAB                   : enabled
20220714 033140.571 INFO             PET3 ESMF_LAPACK                 : enabled
20220714 033140.571 INFO             PET3 ESMF_NETCDF                 : enabled
20220714 033140.571 INFO             PET3 ESMF_PNETCDF                : disabled
20220714 033140.571 INFO             PET3 ESMF_PIO                    : enabled
20220714 033140.571 INFO             PET3 ESMF_YAMLCPP                : enabled
20220714 033140.571 INFO             PET3  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
20220714 033140.571 INFO             PET3  NUMBER_OF_PROCESSORS           4
20220714 033140.571 INFO             PET3  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
20220714 033140.572 INFO             PET3  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 193
20220714 033140.572 INFO             PET3  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
20220714 033140.572 INFO             PET3  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
20220714 033146.601 INFO             PET3  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
20220714 033146.601 INFO             PET3  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
20220714 033146.601 DEBUG            PET3 vmkt_create()#198 Pthread: service=1 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 4194304 bytes
20220714 033146.601 DEBUG            PET3 vmkt_create()#198 Pthread: service=1 (0: actual PET, 1: service thread) - PTHREAD_STACK_MIN: 16384 bytes stack_size: 4194304 bytes
20220714 033146.616 INFO             PET3  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 233
20220714 033149.158 INFO             PET3  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
20220714 033149.158 INFO             PET3  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
20220714 033149.158 INFO             PET3  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
20220714 033149.158 INFO             PET3 Finalizing ESMF
