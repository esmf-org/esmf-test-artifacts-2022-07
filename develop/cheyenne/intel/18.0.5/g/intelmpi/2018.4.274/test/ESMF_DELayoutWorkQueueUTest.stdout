build time -- 2022-06-27 02:57:48.908316
  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
  Beginning Test, file ESMF_DELayoutWorkQueueUTest.F90, line 170
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
  PASS  GridCompCreate() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 187
  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 1
 93
  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 1
 93
  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 1
 93
  PASS  GridCompSetServices() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 1
 93
  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
 <round 1> PET            0  time:   2.620220184326172E-004
 <round 1> PET            3  time:   2.751350402832031E-004
  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
 <round 1> PET            1  time:    4.02878308296204     
  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
  PASS  Run work queue - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 201
 <round 1> PET            2  time:    6.02892494201660     
  PASS  GridCompDestroy() - round 1, ESMF_DELayoutWorkQueueUTest.F90, line 209
  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
  PASS  GridCompCreate() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 217
 *** hi from mygcomp_setvm_withthreads ***
 *** hi from mygcomp_setvm_withthreads ***
 *** hi from mygcomp_setvm_withthreads ***
 *** hi from mygcomp_setvm_withthreads ***
  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
  PASS  GridCompSetVM() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 227
  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 2
 33
  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 2
 33
  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 2
 33
  PASS  GridCompSetServices() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 2
 33
  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
 <round 2> PET            3  time:    2.47610282897949     
  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
 <round 2> PET            2  time:    2.50261807441711     
  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
 <round 2> PET            1  time:    2.50691986083984     
  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
  PASS  Run work queue - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 241
 <round 2> PET            0  time:    2.54237604141235     
  PASS  GridCompDestroy() - round 2, ESMF_DELayoutWorkQueueUTest.F90, line 249
  Ending Test, file ESMF_DELayoutWorkQueueUTest.F90, line 257
  PET           1  Test Elapsed Time    6067.07200000000       msec.
  PET           2  Test Elapsed Time    6070.79400000000       msec.
  PET           3  Test Elapsed Time    6098.08100000000       msec.
  PET           0  Test Elapsed Time    16061.1490000000       msec.
